etna/ciim/constants.py

```diff
@@ -625,3 +625,97 @@ class Display(StrEnum):
 CUSTOM_ERROR_MESSAGES = {
     "invalid_date_range": "There is a problem. Start date cannot be after end date."
 }
+
+TNA_URLS = {
+    "discovery_browse": "https://discovery.nationalarchives.gov.uk/browse/r/h",
+    "tna_accessions": "https://www.nationalarchives.gov.uk/accessions",
+}
+
+#  associate readable names with api identifiers
+ARCHIVE_RECORD_CREATORS_COLLECTION = [
+    {
+        "name": "business",
+        "api_links_indentifier_value": "B",
+        "display_name": "Businesses",
+        "long_display_name": "Businesses",
+    },
+    {
+        "name": "organisation",
+        "api_links_indentifier_value": "O",
+        "display_name": "Organisations",
+        "long_display_name": "Organisations",
+    },
+    {
+        "name": "person",
+        "api_links_indentifier_value": "P",
+        "display_name": "Persons",
+        "long_display_name": "Persons",
+    },
+    {
+        "name": "diary",
+        "api_links_indentifier_value": "D",
+        "display_name": "Diaries",
+        "long_display_name": "Diaries",
+    },
+    {
+        "name": "family",
+        "api_links_indentifier_value": "F",
+        "display_name": "Families",
+        "long_display_name": "Families",
+    },
+    # {"name":"manor", "api_links_indentifier_value": "M", "display_name": "Manors"}, # TODO until API response is ready
+]
+
+
+ARCHIVE_NRA_RECORDS_COLLECTION = [
+    {
+        "name": "paper_catalogue",
+        "display_name": "Paper catalogues",
+        "long_display_name": "Paper catalogues available to view at The National Archives",
+    },
+]
+
+
+@dataclass
+class ContactInfo:
+    address_line1: str = ""
+    address_town: str = ""
+    postcode: str = ""
+    address_country: str = ""
+    map_url: str = ""
+    url: str = ""
+    telephone: str = ""
+    fax: str = ""
+    email: str = ""
+    corresp_addr: str = ""
+    contact_job_title: str = ""
+    contact_title: str = ""
+    contact_first_name: str = ""
+    contact_last_name: str = ""
+
+
+@dataclass
+class FurtherInfo:
+    opening_hours: str = ""
+    holidays: str = ""
+    facilities: List = field(default_factory=list)
+    comments: str = ""
+
+
+@dataclass
+class CollectionInfo:
+    name: str = ""
+    display_name: str = ""
+    long_display_name: str = ""
+    count: int = 0
+    info_list: List = field(default_factory=list)
+
+
+@dataclass
+class ArchiveCollections:
+    collection_info_list: List = field(default_factory=list)
+
+
+@dataclass
+class AccessionsInfo:
+    accession_years: dict = field(default_factory=dict)
```

Comment:
Nice one, and I agree with the puspose of another file, perhaps `data_classes.py` to be more specific. I also think it would fine if were to do this at that same later point, since also we have the following [data classes in constants.py](https://github.com/nationalarchives/ds-wagtail/blob/develop/etna/ciim/constants.py#L31-L67) dataclass-
`class Bucket:, class BucketList: `

Reply:
Yes, but the difference is: `Bucket` and `BucketList` are USED only in constants. Not in other modules.

Reply:
Relocated the new dataclass to etna\records\classes.py since the data is transformed and not used only in constants.